packages/insomnia-inso/src/commands/lint-specification.ts

```diff
@@ -45,7 +45,11 @@ export async function lintSpecification(
       return false;
     }
   } catch (e) {
-    logger.fatal(e.message);
+    let errorMessage = "Failed to load linting specification."
```

Comment:
Fixes 
```
#22 5.321 ./src/commands/lint-specification.ts 48:17-18
#22 5.321 [tsl] ERROR in /packages/insomnia-inso/src/commands/lint-specification.ts(48,18)
#22 5.321       TS2571: Object is of type 'unknown'.
```

Reply:
```suggestion
    let errorMessage = 'Failed to load linting specification.';
```

To fix up the lint error

Reply:
Fixed it, thanks 👍